libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -335,6 +344,15 @@ bool AP_Arming::battery_checks(bool report)
         }
     }
 
+    if (((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_BATTERY)) && 
+        (_min_voltage > 0.0f)) {
+        if (_battery.voltage() < _min_voltage) {
```

Comment:
Alright I can reimplement. I wasn't sure how the battery logic worked, and was wondering if there was a scenario where as long as the battery that was marked primary was fine that we would be happy to arm for flight. But I've never actually tried to use the multiple battery stuff, so I'm fine with changing it to match the behavior you're looking for.